packages/site/src/data/graphql/tools.ts

```diff
@@ -3,10 +3,234 @@ import { Tool } from "@framework/system/src/models/tool"
+	},
+	{
+		name: "Contentful",
+		author: "Contentful",
+		description:
+			"The GraphQL Content API provides a GraphQL API interface to the content from Contentful. Each Contentful space comes with a GraphQL schema based on its content model",
+		image:
+			"https://pbs.twimg.com/profile_images/1554093693501689856/TlwZfp7r_400x400.jpg",
+		href: "https://www.contentful.com/developers/docs/references/graphql/",
+		tags: ["schemas", "frontend", "backend", "development"],
+	},
+	{
+		name: "DatoCMS",
+		author: "DatoCMS",
+		description:
+			"It's the headless CMS for the modern web. More than 25,000 businesses use DatoCMS to create online content at scale from a central hub and distribute it via API.",
+		image:
+			"https://pbs.twimg.com/profile_images/981943986549030915/O7YbftpC_400x400.jpg",
+		href: "https://www.datocms.com/",
+		tags: ["schemas", "frontend", "backend", "development"],
+	},
+	{
+		name: "Strapi",
+		author: "Strapi",
+		description:
+			"By default Strapi create REST endpoints for each of your content-types. With the GraphQL plugin, you will be able to add a GraphQL endpoint to fetch and mutate your content.",
+		image:
+			"https://pbs.twimg.com/profile_images/1513776955186978816/NLZcECc0_400x400.jpg",
+		href: "https://docs.strapi.io/developer-docs/latest/plugins/graphql.html",
+		tags: ["schemas", "frontend", "backend", "development"],
+	},
+	{
+		name: "WunderGraph",
+		author: "WunderGraph",
+		description:
+			"Declaratively turn your services, databases and 3rd party APIs into a secure, unified, and extensible API. Compose, integrate, ship.",
+		image:
+			"https://pbs.twimg.com/profile_images/1588538504971272192/MGxcr3cH_400x400.jpg",
+		href: "https://wundergraph.com/",
+		tags: ["schemas", "frontend", "backend", "development"],
+	},
+	{
+		name: "Fauna",
+		author: "Fauna",
+		description: "Instantly execute GraphQL queries on your data",
+		image: "https://avatars.githubusercontent.com/u/1477000?s=200&v=4",
+		href: "https://fauna.com/graphql",
+		tags: ["schemas", "frontend", "backend", "development"],
+	},
+	{
+		name: "Graphene-Python",
+		author: "graphene-Python",
+		description: "GraphQL in Python Made Easy",
+		image: "https://avatars.githubusercontent.com/u/15002022?s=200&v=4",
+		href: "https://graphene-python.org/",
+		tags: ["schemas", "frontend", "backend", "development"],
+	},
+	{
+		name: "AWS AppSync",
+		author: "AWS",
+		description:
+			"Accelerate application development with serverless GraphQL and Pub/Sub APIs",
+		image:
+			"https://upload.wikimedia.org/wikipedia/commons/thumb/9/93/Amazon_Web_Services_Logo.svg/1024px-Amazon_Web_Services_Logo.svg.png?20170912170050",
+		href: "https://aws.amazon.com/appsync/",
+		tags: ["schemas", "frontend", "backend", "development"],
+	},
+	{
+		name: "Apollo Client Devtools",
+		author: "Apollo",
+		description:
+			"Apollo Client Devtools is a Chrome extension for the open-source GraphQL client, Apollo Client.",
+		image:
```

Comment:
This image appears to be broken. Please update it.

Screenshot:
![Screen Shot 2022-11-08 at 4 27 12 PM](https://user-images.githubusercontent.com/20880360/200706072-66fb80bc-c6d0-4919-9fdb-8bbf1dff04fd.png)